test/parallel/test-http-outgoing-internal-headernames.js

```diff
@@ -0,0 +1,22 @@
+// Flags: --expose-internals
```

Comment:
I don't think we have to expose internals here.

Reply:
Removed.